docs/admin-invitations/0.1/README.md

```diff
@@ -13,151 +10,204 @@ a new DID to be provisioned for the invitation.
 - This version lacks paging of invitation lists.
 - This version lacks a way to filter invitations when listing.
 - The first example for each protocol message is for general usage; the second example specific to the ACA-Py Plugin Toolbox
-
+- Invitation create method will be changed slightly; the user will be able to specify whether to use Out-Of-Band or Connections protocol
```

Comment:
This technically isn't true; we have two separate messages for the two different invitation types rather than a single one where the user can specify one type or the other.